### PR TITLE
Gibbon website: add publications for 2024 Q1,2

### PR DESCRIPTION
**Pull request by ulysses4ever**
Preview: https://ulysses4ever.github.io/gibbon/#publications

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -46,25 +46,18 @@ For more options:
...
-</div>
+## Publications
+
+|||
+|--- |--- |
+|ECOOP'17|**Compiling Tree Transforms to Operate on Packed Representations:** <br/> Michael Vollmer, Sarah Spall, Buddhika Chamith, Laith Sakka, Chaitanya Koparkar, Milind Kulkarni, Sam Tobin-Hochstadt, Ryan Newton [[PDF][ecoop17]]|
+|PLDI'19|**LoCal: A Language for Programs Operating on Serialized Data:** <br/> Michael Vollmer, Chaitanya Koparkar, Mike Rainey, Laith Sakka, Milind Kulkarni, Ryan R. Newton [[PDF][pldi19]]|
+|ICFP'21|**Efficient Tree-Traversals: Reconciling Parallelism and Dense Data Representations:** <br/> Chaitanya Koparkar, Mike Rainey, Michael Vollmer, Milind Kulkarni, Ryan R. Newton [[PDF][icfp21]]|
+|ISMM'24|**Garbage Collection for Mostly Serialized Heaps:** <br/> Chaitanya Koparkar, Vidush Singhal, Aditya Gupta, Mike Rainey, Michael Vollmer, Artem Pelenitsyn, Sam Tobin-Hochstadt, Milind Kulkarni, Ryan R. Newton [[PDF][ismm24]]|
+|ECOOP'24|**Optimizing Layout of Recursive Datatypes with Marmoset:** <br/> Vidush Singhal, Chaitanya Koparkar, Joseph Zullo, Artem Pelenitsyn, Michael Vollmer, Mike Rainey, Milind Kulkarni, Ryan R. Newton [[PDF][ecoop24]]|
```
Comment (vidsinghal) [inline index.md:57]: Fix author order for ECOOP 24

**Review by vidsinghal — CHANGES_REQUESTED** [review]

File: index.md (hunk 1 above)
Comment (ulysses4ever) [reply index.md:57]: What's wrong with the author order?

Comment (vidsinghal) [reply index.md:57]: check preprint: https://arxiv.org/pdf/2405.17590. MK is last not RN.

**Review by vidsinghal — APPROVED** [review]

File: index.md (hunk 1 above)
Comment (ulysses4ever) [reply index.md]: My bad! Fixed, thank you.

**Pull request merged**